creusot/tests/should_fail/bug/specialize.mlcfg

```diff
@@ -277,6 +276,7 @@ module Specialize_F
     goto BB2
   }
   BB2 {
+    assume { Resolve0.resolve v_1 };
```

Comment:
Why are we resolving `v_1` here? it should be unitialized?

Reply:
Same here, we drop `_1` which makes it become initialized.